src/display/text_layer.js

```diff
@@ -186,7 +187,8 @@ var renderTextLayer = (function renderTextLayerClosure() {
       return;
     }
 
-    var canvas = document.createElement('canvas');
```

Comment:
revert the change here -- it is used to measure text length in the DOM, you can add a comment that CanvasRactory is not needed here